test/grouping.jl

```diff
@@ -318,9 +318,13 @@ module TestGrouping
                        c = Vector{Union{Float64, Missing}}(randn(8)))
 
         cols = [:a, :b]
-        f(df) = DataFrame(cmax = maximum(df[:c]))
-        g(df) = (cmax = maximum(df[:c]),)
-        h(df) = maximum(df[:c])
+        f(df) = DataFrame(cmax = maximum(df[:, :c]))
```

Comment:
https://github.com/JuliaData/DataFrames.jl/pull/1592 will fix this so better leave it out from this PR (thanks for spotting it!).

Reply:
OK